lib/middleware/html_response_transformer.rb

```diff
@@ -32,6 +33,7 @@ def transform(response_body)
       response_body = ResponsiveImages.new(response_body).html
       response_body = LazyLoadImages.new(response_body).html
       response_body = ExternalLinks.new(response_body).html
+      response_body = ErrorTitle.new(response_body).html
```

Comment:
Just wondering ... is this not starting to get expensive to parse and re-parse the document 6 times with Nokogiri?

Reply:
(I feel like I've asked this question on a previous PR ... the one for the LazyLoadImages maybe?)

Reply:
Looking into Skylight it looks like this method may be taking up to ~1/3 of the request time ... we agreed to have a look at that as a separate piece of work.